src/jepsen/voltdb.clj

```diff
@@ -194,19 +195,18 @@
 (defn start-daemon!
   "Starts the VoltDB daemon."
   [test]
-  (c/sudo username
           (c/cd base-dir
                 (info "Starting voltdb")
-                (cu/start-daemon! {:env {:JAVA_HOME "/opt/oracle_java17" :PATH "/opt/oracle_java17/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin" }
+                (cu/start-daemon! {:env {:JAVA_HOME (System/getenv "JAVA_HOME") }
                                    :logfile (str base-dir "/log/stdout.log")
                                    :pidfile pidfile
                                    :chdir   base-dir}
-                                  (str base-dir "/bin/voltdb")
+                                  "bin/voltdb"
```

Comment:
is "." in the PATH? otherwise we might not find "./bin/voltdb" from the directory base-dir.

Reply:
I don't have "." in my path and it works, so jepsen is automatically adding it.